service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/OrderIssuedEmailContentProvider.java

```diff
@@ -0,0 +1,88 @@
+import java.util.Map;
+
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+import static uk.gov.hmcts.reform.fpl.enums.IssuedOrderType.CMO;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildSubjectLine;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildSubjectLineWithHearingBookingDateSuffix;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
+import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.formatRepresentativesForPostNotification;
+import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
+import static uk.gov.service.notify.NotificationClient.prepareUpload;
+
+@Slf4j
+@Service
+public class OrderIssuedEmailContentProvider extends AbstractEmailContentProvider {
+
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
+    private final RepresentativeService representativeService;
+    private final ObjectMapper objectMapper;
+
+    public OrderIssuedEmailContentProvider(@Value("${ccd.ui.base.url}") String uiBaseUrl,
+                                           ObjectMapper objectMapper,
+                                           HearingBookingService hearingBookingService,
+                                           LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration,
+                                           DateFormatterService dateFormatterService,
+                                           RepresentativeService representativeService) {
+        super(uiBaseUrl, dateFormatterService, hearingBookingService);
+        this.objectMapper = objectMapper;
+        this.representativeService = representativeService;
+        this.localAuthorityNameLookupConfiguration = localAuthorityNameLookupConfiguration;
+    }
+
+    public Map<String, Object> buildOrderNotificationParametersForHmctsAdmin(final CaseDetails caseDetails,
+                                                                             final String localAuthorityCode,
+                                                                             final byte[] documentContents,
+                                                                             final IssuedOrderType issuedOrderType) {
+        CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
+        final String cmoCallout = buildSubjectLine(caseData);
+        List<Representative> representativesServedByPost = representativeService.getRepresentativesByServedPreference(
+            caseData.getRepresentatives(), POST);
+        List<String> formattedRepresentatives = formatRepresentativesForPostNotification(representativesServedByPost);
+
+        return ImmutableMap.<String, Object>builder()
+            .put("cmoCallout", issuedOrderType == CMO ? "^" + buildSubjectLineWithHearingBookingDateSuffix(
```

Comment:
```suggestion
            .put("callout", issuedOrderType == CMO ? "^" + buildSubjectLineWithHearingBookingDateSuffix(
```

Reply:
not fussed but cmoCallout adds no benefit

Reply:
thought it might help people looking at gov notify template in future to know that the callout is only shown for CMO (since same template handles 3 types of order). think there is benefit